### ISSUE TITLE
Add bibliographic citation as download to DOI pages  

### ISSUE THREAD

**Issue by kcopas** [gbif/portal-feedback#2027]
Building on #1916 , here's a suggestion made during a SPNHC session

Add an option to download a bibliographic text file of the full search/download-DOI citation to the DOI page. Likeliest flavours include RIS (.ris), BibTeX (.bib), RDF or EndNote XML (.xml).

**Comment by MortenHofft** [comment]
This has also been suggested by @dschigel and myself over the last years. Labelling as API idea.

**Comment by mdoering** [comment]
what about [CSL-JSON](https://github.com/citation-style-language/schema#csl-json-schema)? Afaik this is the most modern of the standards

**Comment by dschigel** [comment]
Yes, still very much in favor of this functionality. I like how this works in Google Scholar: click **"** below any search result and see five text variants and four export options. If course, ours should have DOI tatooed into all variants and exports.
